Felienne/hedy
# [FIX] Only highlighting username on program view page when public profile

**Description**
In this PR we fix a small issue where the username on the program view page was always click-able, independent of the user actually has a public profile. This is fixed to better align with the `/explore` page where the username is only click-able iff there is a related public profile.

**Fixes**
This PR fixes #2408

**How to test**
Make sure you have an account with a related public profile. Share a program, navigate to it through the `/explore` page and verify that the username is indeed click-able, correctly re-directing you to the profile. Now when removing the public profile the username should no longer be click-able.
